Userland/Shell/Builtin.cpp

```diff
@@ -939,6 +1010,8 @@ bool Shell::run_builtin(const AST::Command& command, const NonnullRefPtrVector<A
         retval = builtin_##builtin(argv.size() - 1, argv.data());        \
         if (!has_error(ShellError::None))                                \
             raise_error(m_error, m_error_description, command.position); \
+        fflush(stdout);                                                  \
+        fflush(stderr);                                                  \
```

Comment:
I wish you had put these in a separate commit...or at least mentioned them in the description of this commit :P